src/Asset/ManifestLookup.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
```

Comment:
Should we declare classes as strict?

Reply:
I'm fine with that, but we're not using them in `symfony/symfony` yet - I'm not sure if that's a policy, or for other reasons (that don't apply here)

Reply:
Personally I don't see the benefit of strict mode: it provides no benefit to external APIs
of course strict mode can help spot some errors inside the boundaries of the facing API, but anyway this is covered by community work+tests.

Reply:
@nicolas-grekas 

> it provides no benefit for the "public" facing APIs.

Doesn't it? Does it not trigger type errors if the API is misused from userland?

Reply:
Nope, not any, see also discussion in https://github.com/symfony/symfony/issues/28423

Reply:
You're right, I misunderstood this comment. :see_no_evil: Thanks!

Reply:
triggering such error is what types are doing, not strict mode (strict mode in *userland* makes type stricter for arguments)